tests/SprykerEcoTest/Zed/ArvatoRss/Business/Api/Converter/RiskCheckRequestConverterTest.php

```diff
@@ -123,4 +125,24 @@ protected function testResult(array $result, ArvatoRssRiskCheckRequestTransfer $
             );
         }
     }
+
+    /**
+     * @param ArvatoRssRiskCheckRequestTransfer $requestTransfer
+     * @param RiskCheckRequestConverter $converter
+     */
+    protected function testDeliveryPresents(ArvatoRssRiskCheckRequestTransfer $requestTransfer, RiskCheckRequestConverter $converter)
+    {
+        $result = $converter->convert($requestTransfer);
+        $this->assertArrayHasKey(ArvatoRssRequestApiConfig::ARVATORSS_API_DELIVERYCUSTOMER, $result);
+
+        $requestTransfer = (new ArvatoRssRiskCheckRequestBuilder())
```

Comment:
Please create separate tests.
Each test should test one case/function.
Consider 'dataProvider' usage in case the code is similar